source

```diff
@@ -66141,6 +66142,36 @@ dictionary <dfn>ElementDefinitionOptions</dfn> {
    <span>connected</span>.</dd>
   </dl>
 
+  <p>New instances of <code>CustomElementRegistry</code> can be defined and attached to a
+  <span>ShadowRoot</span> through the registry option from <code
+  data-x="dom-Element-attachShadow">attachShadow()</code>. Those instances are are independent of
+  the instance available at <code data-x="dom-window-customElements">window.customElements</code>,
+  as the latter is attached to the
+  <span data-x="concept-document-window">associated <code>Document</code></span>.</p>
+
+  <dl class="domintro">
+    <dt><var>registry</var> = new <code data-x="dom-CustomElementRegistry">CustomElementRegistry</code>()</dt>
+
+    <dd>Defines a new <span data-x="dom-CustomElementRegistry">Registry</span> that can map an
+    independent set of <span data-x="dom-window-customElements">custom elements</span>, which is
+    not attached yet to any <span>Document</span> or any <span>ShadowRoot</span>.</dd>
```

Comment:
I believe custom registries can only be associated to a shadow root instance, not to a document.

also, I will like to use the the word "association" rather than "attach", because that opens the door to change that association at any given time.